signup.js

```diff
@@ -30,13 +30,16 @@ form.addEventListener("submit", async (e) => {
   const json = await response.json();
 
   if (json.error === true) {
+    //prendere il messaggio generico di errore
+    const message = json.message;
+    // alert(message);
     // Get the json value given by the backend as an array of array
     const validation = json.validations;
     //[
     //[ "name",["Name is required", "Name should have only letter",..] ],
     //[ "password",["Name is required", "Name should have only letter",..] ]
     // ]
-    const messagesArray = [];
+    const messagesArray = [message + ":" + ""];
```

Comment:
Why `message + ":" + ""` ?

Reply:
era per concatenare (in maniera sbagliata la stringa del messaggio generico ai messaggi di validation ....
in realtá ho cambiato questa cosa e ho anche escluso il nome dell'utente in corso dalla lista degli invitati (ma senza usare il filter). pusho ??